code/modules/migrations/SS13/011-port-library-to-feedback-db.dm

```diff
@@ -0,0 +1,30 @@
+/datum/migration/mysql/ss13/_011
+	id = 11
+	name = "Library in feedback DB"
+
+/datum/migration/mysql/ss13/_011/up()
+	if(!hasTable("library"))
+		return execute({"CREATE  TABLE IF NOT EXISTS `tgstation`.`library` (
```

Comment:
Extra space between CREATE and TABLE. Also our db is called feedback, not tgstation

Reply:
Got it. I think I can even remove the table name, as we're going to only get one anyways.